backend/src/main/java/com/ody/meeting/controller/MeetingControllerSwagger.java

```diff
@@ -90,4 +92,34 @@ ResponseEntity<MeetingSaveResponse> save(
     @ErrorCode401
     @ErrorCode500
     ResponseEntity<Void> validateInviteCode(@Parameter(hidden = true) Member member, String inviteCode);
+
+    @Operation(
+            summary = "약속 참여자 eta 상태 목록 조회",
+            requestBody = @RequestBody(content = @Content(schema = @Schema(implementation = MateStatusRequest.class))),
+
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "약속 참여자 eta 상태 목록 조회 성공",
+                            content = @Content(schema = @Schema(implementation = MateStatusResponses.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "클라이언트 입력 오류 또는 약속 시간 30분 이전 조회 시도 시",
```

Comment:
[제안] '약속 시간 30분 이전'이라는 표현이 의도가 잘 전달될 것 같은가요? 노션에다가 제가 쓴 문장인데 작성 당시에 살짝 어색했어서 수정을 제안해봅니다..

```suggestion
                            description = "클라이언트 입력 오류 또는 약속 시간 30분 전보다 이른 시간에 조회 시도 시",
```

Reply:
"이전"도 좋다고 생각하는데 제리가 제안한 대로 변경해주면 좀 더 직관적으로 와닿을 꺼 같아요 !
반영완료 👍🏻